Python/Product/BuildTasks/TestTargets/Microsoft.PythonTools.Django.targets

```diff
@@ -20,12 +20,7 @@ permissions and limitations under the License.
   <PropertyGroup>
     <!-- Enable logging for publish/preview -->
     <EnablePackageProcessLoggingAndAssert>true</EnablePackageProcessLoggingAndAssert>
-    <!-- Allow override; fallback to current VS install root -->
-    <_VSVer Condition="'$(VisualStudioVersion)' == ''">18.0</_VSVer>
-    <PythonToolsTargetsDir Condition="'$(PythonToolsTargetsDir)' == ''">$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v$(_VSVer)\Python Tools</PythonToolsTargetsDir>
   </PropertyGroup>
 
-  <!-- Only import if an updated Django targets file actually exists -->
-  <Import Project="$(PythonToolsTargetsDir)\Microsoft.PythonTools.Django.Core.targets"
-          Condition="Exists('$(PythonToolsTargetsDir)\Microsoft.PythonTools.Django.Core.targets')" />
+  <Import Project="$(_PythonToolsPath)\..\..\Python - Django\$(VisualStudioVersion).0\Microsoft.PythonTools.Django.targets" />
```

Comment:
this path is restoring the previous logic before i changed it.. except to get 18.0.0 or 17.0.0 we use '$(VisualStudioVersion).0'

Reply:
always there